src/sagas/publicChannels/publicChannels.slice.ts

```diff
@@ -137,42 +140,56 @@ export const publicChannelsSlice = createSlice({
       state,
       action: PayloadAction<GetPublicChannelsResponse>
     ) => {
-      console.log(`replicated channels [${Object.keys(action.payload.channels)}] for community ${action.payload.communityId}`)
-      log(action.payload.channels)
-      log({...state.entities})
-      channelsByCommunityAdapter.updateOne(state, {
+      console.log(
+        `replicated channels [${Object.keys(
+          action.payload.channels
+        )}] for community ${action.payload.communityId}`
+      );
+      log(action.payload.channels);
+      log({ ...state.channels.entities });
```

Comment:
lefover?

Reply:
It's highlighted because of prettier. It was already here. Should I get rid of that?